File: app/src/main/java/com/BlizzardArmory/starcraft/Profile/BestTeamFinish.java
package com.BlizzardArmory.starcraft.Profile;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class BestTeamFinish {

    @SerializedName("leagueName")
    @Expose
    private String leagueName;
    @SerializedName("timesAchieved")
    @Expose
    private int timesAchieved;

    public String getLeagueName() {
        return leagueName;
    }

    public void setLeagueName(String leagueName) {
        this.leagueName = leagueName;
    }

    public int getTimesAchieved() {
        return timesAchieved;
    }

    public void setTimesAchieved(int timesAchieved) {
        this.timesAchieved = timesAchieved;
    }

}

File: app/src/main/java/com/BlizzardArmory/model/starcraft/profile/BestTeamFinish.kt
package com.BlizzardArmory.model.starcraft.profile

import androidx.annotation.Keep
import com.google.gson.annotations.SerializedName


/**
 * The type Best team finish.
 */
@Keep
data class BestTeamFinish(

    @SerializedName("leagueName")
    var leagueName: String?,

    @SerializedName("timesAchieved")
    var timesAchieved: Int

)